tailwind.config.ts

```diff
@@ -0,0 +1,20 @@
+import type { Config } from "tailwindcss";
+
+const config: Config = {
+  content: [
+    "./app/**/*.{js,ts,jsx,tsx,mdx}",
+    "./packages/**/*.{js,ts,jsx,tsx,mdx}",
+  ],
+  theme: {
+    extend: {
+      colors: {
+        "gray-x-500": "#718096",
+        "purple-x-400": "#9f7aea",
```

Comment:
No need to extend the default theme unless absolutely necessary. Each change we make must be maintained long-term.

Reply:
Made in https://github.com/podcodar/webapp/pull/173/commits/8b6379adb4761c9cad88a8163ed982a33abe08b8